src/NuGet.Services.V3/Registration/Models/RegistrationCatalogEntry.cs

```diff
@@ -75,5 +75,9 @@ public class RegistrationCatalogEntry
 
         [JsonProperty("version")]
         public string Version { get; set; }
+
+        [JsonProperty("vulnerabilities")]
+        public List<PackageVulnerability> Vulnerabilities { get; set; }
```

Comment:
It's better to be explicit with the type. If/when we ever move to System.Text.Json, that doesn't support polymorphic serialization.

```suggestion
        public List<RegistrationPackageVulnerability> Vulnerabilities { get; set; }
```